src/CommonITILCost.php

```diff
@@ -63,7 +63,7 @@ public function getTabNameForItem(CommonGLPI $item, $withtemplate = 0)
 
        // can exists for template
         if (
-            ($item->getType() == static::$itemtype)
+            (get_class($item) == static::class)
```

Comment:
We want to check that item correspond to the linked type, not to the current class.
```suggestion
            (get_class($item) == static::$itemtype)
```